scripts/us_census/pep/monthly_population_estimate/download_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to automate the testing for USA Population preprocess script.
+"""
+
+import os
+import unittest
+from os import path
+from download import _download
+import pandas as pd
+# module_dir_ is the path to where this test is running from.
+module_dir_ = os.path.dirname(__file__)
+test_data_folder = os.path.join(module_dir_, "test_data")
+op_data_folder = os.path.join(module_dir_, "test_output_data_download")

```

Comment:
python type nit... for file-local globals please style as _OP_DATA_FOLDER.

Reply:
made the style as protected with Capitals wherever necessary